### DISCUSSION TITLE
One-line installer lead to backing up kernel, and preparing for a new build of OS from SRC??? Am I Hallucinating?

### DISCUSSION THREAD

**Discussion by anoduck** [Slackadays/Clipboard#149] [Q&A]
I don't know how this happened or why, and perhaps someone can shed some light onto the experience. 

Ran the one-line installer:

```bash
curl -sSL https://github.com/Slackadays/Clipboard/raw/main/install.sh | sh
```

Output was:
```bash
|>$ curl -sSL https://github.com/Slackadays/Clipboard/raw/main/install.sh | sh
Fetching bsds
Using bsd.mp..
Backed up the following:
	/bsd => /obsd
	/bsd.rd => /obsd.rd
	/sbin/reboot => /sbin/oreboot
(SHA256) bsd: OK
(SHA256) bsd.mp: OK
(SHA256) bsd.rd: OK
^C%
```

I have read the install script, and don't see anything to cause this output. Which is very very concerning.

**Comment by Slackadays** [comment]
What system are you using? Because it's possible that you're using something like Gentoo and you have an old version, so Emerge wants to compile everything or something like that

Other than that, I don't know what might possibly cause what you're seeing

**Reply by anoduck** [reply]
I know, it was suspiciously weird as all get out. I am running OpenBSD, so it is an enigma, and is still occurring.

**Reply by anoduck** [reply]
@Slackadays I just ran the installation script on my development Linux box. It really doesn't like running in ZSH, and if it isn't run with `sudo` becomes caught in a loop during authorization.
